skyrim-platform/src/platform_se/skyrim_platform/CallNative.cpp

```diff
@@ -111,45 +111,53 @@ class VariableAccess : public RE::BSScript::Variable
 };
 
 namespace {
-CallNative::AnySafe VariableToAnySafe(
+
+CallNative::ObjectPtr GetOneObjectPtr(
```

Comment:
Really minor stuff, but I feel like `GetSingleObjectPtr` would be a bit better here (one/many vs single/multiple). Not really a request to change

Reply:
Do you mean place it after `VariableToAnySafe`?

Reply:
Nope, placement is totally ok here (at least from my point of view). I meant just renaming:

```suggestion
CallNative::ObjectPtr GetSingleObjectPtr(
```

IMO 'Single' sounds kinda more appropriate. But that's questionable

Reply:
Oh.. okey, ofccourse...